src/client/components/veteran-information/BirthInformationSection.jsx

```diff
@@ -55,6 +55,7 @@ class BirthInformationSection extends React.Component {
               name="cityOfBirth"
               field={this.props.data.cityOfBirth}
               autocomplete="off"
+              charMax={20}
```

Comment:
From the spreadsheet I've been looking at, 'Place of Birth' city can be up to 20, 'Current' City can be up to 51, 'Next of Kin' city can be up to 30, 'Emergency Contact' can be 30, 'Insurance Company' city can be 25, 'Employment' city can be 30. Cities are just all over the place.